### ISSUE TITLE
Buildkite trigger step is not supported properly

### ISSUE THREAD

**Issue by mowies** [ksindi/buildpipe#39]
I want to use buildpipe to detect changes and to trigger different buildkite pipelines through buildkite's built-in [`trigger` step](https://buildkite.com/docs/pipelines/trigger-step)

But the syntax that buildpipe produces seems to be wrong for this step.
The trigger step does not have the `env` attribute on its top level but only under the `build` attribute.

Step produced by buildpipe on my pipeline:
```
- async: true
  build:
    branch: $BUILDKITE_BRANCH
    commit: $BUILDKITE_COMMIT
    message: $BUILDKITE_MESSAGE
  env:
    BUILDPIPE_STAIR_NAME: trigger-<mypipeline>
    BUILDPIPE_STAIR_SCOPE: stair
  label: <mylabel>
  trigger: <slug of a different pipeline>
```

How it should look like:
```
- async: true
  build:
    branch: $BUILDKITE_BRANCH
    commit: $BUILDKITE_COMMIT
    message: $BUILDKITE_MESSAGE
    env:
      BUILDPIPE_STAIR_NAME: trigger-<mypipeline>
      BUILDPIPE_STAIR_SCOPE: stair
  label: <mylabel>
  trigger: <slug of a different pipeline>
```

**Comment by ksindi** [comment]
Thanks for the find @mowies! Do you think you can make a PR for the fix? Otherwise I might have time this weekend.

**Comment by mowies** [comment]
I started working on it today.

**Comment by mowies** [comment]
Solved in #41
